### PR TITLE
Update blob tutorial to remove runtime errors and improve marker visi…

### PR DESCRIPTION
**Pull request by Azrun**
[bug]

## About the PR 
Addresses #371 
Add outline to marker in blob tutorial to improve visibility. (Hue shifting marker was considered but didn't want to learn how to properly apply color matrix).
Allows completed/exited tutorials to be added back into Blob tutorial pool.
Overlays usage to key mapping as per UpdateOverlays() 

## Why's this needed? 
Improves Blob tutorial

## Changelog

```
(u)Azrun:
(+)Blob tutorial marker visibility improved.  Should now be able to complete and re-attempt tutorial ad infinitum.
```

### CODE REVIEW

File: code/modules/tutorials/blob_tutorial.dm [hunk 1]
```diff
@@ -735,7 +736,7 @@ proc/AddBlobSteps(var/datum/tutorial/blob/T)
 	T.AddStep(new /datum/tutorialStep/blob/cutscene3)
 	T.AddStep(new /datum/tutorialStep/blob/finished)
 
-/obj/blob_tutorial_walker
+/mob/blob_tutorial_walker
```
Comment (ZeWaka) [inline code/modules/tutorials/blob_tutorial.dm:739]: why?

Comment (Azrun) [reply code/modules/tutorials/blob_tutorial.dm:739]: Didn't see why flamethrowers should handle being invoked by arbitrary objects. 

Alternatives:
A) Adding a property to that object.
B) Gracefully handle lack of object property at caller.

Comment (ZeWaka) [reply code/modules/tutorials/blob_tutorial.dm:739]: It's just weird creating an entirely new mob basetype just for this, furthermore it's not like it uses any mob functionality.

Comment (Tarmunora) [reply code/modules/tutorials/blob_tutorial.dm:739]: shouldn't be hard to add some checks in flamer code to handle being fired without `mob/user`

Comment (Azrun) [reply code/modules/tutorials/blob_tutorial.dm]: Thanks! Ended up turning into a great lesson in type safety...

**Review by ZeWaka — APPROVED** [review]
btw feel free to resolve comments when you address them

**Pull request merged**